packages/flutter/test/cupertino/dialog_test.dart

```diff
@@ -12,1237 +12,2225 @@ import '../rendering/mock_canvas.dart';
 import '../widgets/semantics_tester.dart';
```

Comment:
It's fine to keep the tests in two separate files. In fact that's probably better -- ideally this PR wouldn't touch the tests, that's the best way to show that it didn't break anything.